.changeset/breezy-books-visit.md

```diff
@@ -0,0 +1,68 @@
+---
+"@quiltt/react-test-nextjs": minor
+"@quiltt/react": minor
+"@quiltt/core": minor
+---
+
+This introduces a new Javascript API that can be used instead of or with the DOM API, giving exposure to exit events. There are a few ways to use it:
+
+## 1. HTML5
+
+If you're using the HTML interface, and need to upgrade to using some Javascript code, you can; but all event registrations are on a global level. This means that if you have multiple buttons, you will look at the metadata of the response to see which one you're reacting to.
+
+```html
+<head>
+  <script src="http://cdn.lvh.me:3000/v1/connector.js"></script>
+  <script language="JavaScript">
+    Quiltt.onExitSuccess((metadata) =>
+      console.log("Global onExitSuccess", metadata.connectionId)
+    );
+  </script>
+</head>
+<body>
+  <button quiltt-button="<CONNECTOR_ID">Click Here!</button>
+</body>
+```
+
+## 2. Javascript
+
+Now if you want to do something more complex, and expect to be working with multiple buttons in different ways, then the Javascript SDK may be the way to go. With this, you can control everything in JS.
+
+```html
+<head>
+  <script src="http://cdn.lvh.me:3000/v1/connector.js"></script>
```

Comment:
Should this example show the production CDN?

Reply:
Good catch - we need to fix it. Thanks